### PR TITLE
fix: refactor setup

### PR DESCRIPTION
**Pull request by livio-a**

### CODE REVIEW

File: internal/setup/step1.go [hunk 1]
```diff
@@ -1,52 +1,53 @@
...
-func (s *Step1) step() iam_model.Step {
-	return iam_model.Step1
-}
-
-//func (s *Step1) init(setup *Setup) {
-//	s.setup = setup
-//	s.createdUsers = make(map[string]*usr_model.User)
-//	s.createdOrgs = make(map[string]*org_model.Org)
-//	s.createdProjects = make(map[string]*proj_model.Project)
+//
```
Comment (fgerschwiler) [inline internal/setup/step1.go:3]: remove file if not used anymore

File: internal/v2/command/setup.go [hunk 2]
```diff
@@ -25,6 +72,27 @@ func (r *CommandSide) StartSetup(ctx context.Context, iamID string, step domain.
 	return writeModelToIAM(iamWriteModel), nil
 }
 
+func (r *CommandSide) setup(ctx context.Context, step Step, iamAggregateProvider func(*IAMWriteModel) (*iam_repo.Aggregate, error)) error {
+	iam, err := r.iamByID(ctx, r.iamID)
+	if err != nil && !caos_errs.IsNotFound(err) {
+		return err
+	}
+	if iam.SetUpStarted != step.Step() && iam.SetUpDone+1 != step.Step() {
+
```
Comment (fgerschwiler) [inline internal/v2/command/setup.go:81]: Empty if? it not used remove, else add todo

File: internal/setup/step1.go (hunk 1 above)
Comment (livio-a) [reply internal/setup/step1.go]: added todo

File: internal/v2/command/setup.go (hunk 2 above)
Comment (livio-a) [reply internal/v2/command/setup.go]: fixed

**Review by fgerschwiler — APPROVED** [review]

**Pull request merged**